src/CRM/CivixBundle/Resources/views/Code/test-legacy.php.php

```diff
@@ -22,33 +22,42 @@
  */
 class <?php echo $testClass ?> extends \CiviUnitTestCase implements HeadlessInterface {
 
-  public function setUpHeadless() {
-    // `CiviTestListener`+`HeadlessInterface` has some clever tricks for
-    // bootstrapping which make it easier to use phpunit CLI.
-    // However, there's not much point in using setupHeadless() with CiviUnitTestCase
-    // because CiviUnitTestCase performs its own special setup/teardown logic.
-  }
+  /**
+   * Setup for when Headless interface is implemented.
+   *
+   * `CiviTestListener`+`HeadlessInterface` has some clever tricks for
+   * bootstrapping which make it easier to use phpunit CLI.
+   * However, there's not much point in using setupHeadless() with CiviUnitTestCase
+   * because CiviUnitTestCase performs its own special setup/teardown logic.
+   */
+  public function setUpHeadless(): void {}
```

Comment:
Doesn't make sense on a few fronts:

* This disagrees with `HeadlessInterface`'s signature for `setUpHeadless()`
* Other templates have it return a value...